app/src/sharedTest/java/org/oppia/app/topic/overview/TopicOverviewFragmentTest.kt

```diff
@@ -25,13 +29,33 @@ import javax.inject.Singleton
 @RunWith(AndroidJUnit4::class)
 class TopicOverviewFragmentTest {
```

Comment:
Suggest also adding tests to verify that the correct image is shown, and that the correct description is shown.

Reply:
For checking the image, I have introduced a custom `DrawableMatcher`. @BenHenning you will have to have a look at this implementation.

Reply:
This looks really reasonable, thanks @rt4914!